molecule/default/requirements.yml

```diff
@@ -3,9 +3,5 @@
   name: cyhy_core
 - src: https://github.com/cisagov/ansible-role-ncats-webd
   name: ncats_webd
-- src: https://github.com/cisagov/ansible-role-pip
-  name: pip
-- src: https://github.com/cisagov/ansible-role-python
-  name: python
```

Comment:
I know this is technically covered by the [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core) dependency but please explicitly add these as role dependencies (with appropriate variable settings) since we are installing the package with `pip2`.

Reply:
Please see commit 468b9d5.